### ISSUE TITLE
Support for additional keyboard layouts

### ISSUE THREAD

**Issue by tejado** [tejado/Authorizer#16]
Would be great if someone creates additional keyboard layouts for Authorizer.

More information are available in PR https://github.com/tejado/Authorizer/pull/3

**Comment by darshan934** [comment]
Are there any specific keyboards that I should add?

**Comment by tejado** [comment]
No. To be honest, I'm also not sure what other keyboards exists. Maybe common ones like Spain etc would be great. But I'm happy about every additional one =)

**Comment by papagon** [comment]
Hello,
5 years later, I'm reopening the discussion.
I would be very interested in integrating the native layout on my computer (France), French (FR).
The application already offers French (CH), but this keyboard has little to do with French Azerty.
Thanks a lot!

**Comment by tejado** [comment]
Hi @papagon 
could you do a PR? Or would you need help how to find out the correct mappings?

**Comment by papagon** [comment]
Hi @tejado 
I've found a existing mapping https://github.com/hid-io/layouts/tree/master for my keyboard.
I'll try to map it and do a PR !

**Comment by papagon** [comment]
@tejado 
Done :-) Unfortunately, as I don't have anything to build the APK, I wasn't able to test it.
